Resources/Locale/en-US/cargo/bounties.ftl

```diff
@@ -100,3 +117,20 @@ bounty-description-warm-cloth = The Unath construction crew freezes and is unabl
 bounty-description-battery = As the Arachnid settlement prepares for a solar flare, they are requesting a large shipment of power batteries. We're sending out a request for delivery.
 bounty-description-lasergun = The Salvage Caravan requests a large shipment of laser weapons to mop up a hive of xenomorphs.
 bounty-description-food = After the rat king invasion, a neighboring unathi station was left completely without food. A large meat food shipment is needed.
+bounty-description-fruit = A heroic monkey helped the chaplain catch a troublemaker hiding in the chapel, and the crew wants to reward him for his good work.
+bounty-description-vegetable = The new chef is a vegetarian, and botany can't keep up with their demands. We need some additional veggies to help keep things stocked.
+bounty-description-chili = Today's the Centcomm Chili Cookoff, and, well, a few of us forgot to make some. Please help cover for us.
+bounty-description-live-mouse = Hamlet has gone missing, and the captain is blind and deaf enough that he won't notice a... similar replacement. Alive, of course.
```

Comment:
Same. Hamlet is unique

Reply:
Removed the whole thing, checking for alive is a pain